STAR/draft-ietf-acme-star.md

```diff
@@ -220,23 +216,6 @@ The following subsections describe the three main phases of the protocol:
 - Auto-renewal: the ACME CA periodically re-issues the short-term certificate and posts it to a public URL ({{proto-auto-renewal}});
```

Comment:
This is related to our email discussion: is it always a "public URL", or only when specified explicitly in this order?

Reply:
good catch